PlayCover/Views/AppSettingsView.swift

```diff
@@ -21,6 +21,10 @@ struct AppSettingsView: View {
     @State var hasPlayTools: Bool?
     @State var hasAlias: Bool?
 
+    @State private var isSwitchingPlayToolsInjection = false
```

Comment:
Is it possible to combine these into one variable (and it's references below)? Maybe something like `isInBlockingTask`.

Reply:
Yeah, that would be good.

Reply:
Done.